src/python/pants/util/frozendict.py

```diff
@@ -64,6 +64,15 @@ def _freeze(obj):
 
         return cls({k: _freeze(v) for k, v in data.items()})
 
+    @staticmethod
+    def frozen(to_freeze: Mapping[K, V]) -> FrozenDict[K, V]:
+        """Returns a `FrozenDict` containing the keys and values of `to_freeze`.
+
+        If `to_freeze` is already a `FrozenDict`, returns the same object.
+        """
+
+        return to_freeze if isinstance(to_freeze, FrozenDict) else FrozenDict(to_freeze)
+
```

Comment:
this one is so similar to `deep_freeze()`, I wonder if we shouldn't consolidate them into one, maybe with a `deep: bool=False` arg? (fine to punt on this q to another time)